app/views/votes/new.html.erb

```diff
@@ -1,10 +1,9 @@
 <!-- <div class="container"> -->
-  <%= link_to challenges_path do %>
-  <i class="fas fa-chevron-left"></i>
-  <% end %>
-
-
-  <!-- <div class="container voting-page"> -->
+  <div class="back-arrow">
```

Comment:
do you use this class anywhere? can't find it referenced

Reply:
oh, no. decided to make the hamburger smaller rather than the arrow bigger